src/plugin/transform/gemini.test.ts

```
@@ -486,5 +490,979 @@ describe("transform/gemini", () => {
+
+    it("removes const field", () => {
+      const schema = { const: "fixed_value" };
+      const result = toGeminiSchema(schema) as Record<string, unknown>;
+      expect(result).not.toHaveProperty("const");
+    });
+
+    it("removes conditional schema fields (if/then/else/not)", () => {
+      const schema = {
+        type: "object",
+        if: { properties: { type: { const: "a" } } },
+        then: { properties: { a: { type: "string" } } },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)